Source/UI/PaymentForm/View/CardholderName/CardholderView.swift

```diff
@@ -30,9 +37,9 @@ final class CardholderView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 
-    func update(style: CellTextFieldStyle) {
+    func update(style: CellTextFieldStyle, text: String?) {
         self.style = style
-        cardholderInputView.update(style: style)
+        cardholderInputView.update(style: style, textFieldValue: text)
```

Comment:
I would suggest to override the text here 
```suggestion
       if let text = text {
            style.textfield.text = text
        }
        cardholderInputView.update(style: style)
```

Reply:
Cool observation, I knew I must have missed something 🤦‍♂️